radixdlt/src/integration/java/com/radixdlt/integration/recovery/RecoveryLivenessTest.java

```diff
@@ -106,7 +107,10 @@
 	private NodeEvents nodeEvents;
 
 	public RecoveryLivenessTest(int numNodes, long epochCeilingView) {
-		this.nodeKeys = Stream.generate(ECKeyPair::generateNew).limit(numNodes).collect(Collectors.toList());
+		this.nodeKeys = Stream.generate(ECKeyPair::generateNew)
+			.limit(numNodes)
+			.sorted(Comparator.<ECKeyPair, EUID>comparing(k -> k.getPublicKey().euid()).reversed())
```

Comment:
Why is this reversed?

Reply:
I did it for consistency with what we have in `DeterministicTest` [L132]; it may make debugging slightly easier if all tests use the same ordering. Not sure why it was originally reversed there though, it shouldn't matter if it's reversed or  not.

Reply:
Hmm.  Not sure it makes sense in DeterministicTest either, although git tells me I put that code in there ☹️ 

If it doesn't matter if it is reversed or not, why does the ordering matter at all then?

Reply:
Ah, OK.  Then is it worth simplifying this and DeterministicTest by removing the .reversed() ?

Reply:
Probably yes, unless there's any hidden dependency on reversed order in DeterministicTest. I'll give it a try.